### PR TITLE
Feat/site map

### PR DESCRIPTION
**Pull request by tom-karasch**
Doug,
I think I have this correct now. Does this look like what you are thinking for design of page structure?

### CODE REVIEW

**Review by dougwaldron — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | https://localhost:7229 |

    
| Device | URL |
|--|--|
| mobile | https://localhost:7229 |

Not what you expected? Are your scores flaky? [Try running on Foo instead](https://github.com/foo-software/lighthouse-check#foos-automated-lighthouse-check-api-usage)

File: docs/Site map.md [hunk 1]
```diff
@@ -1,22 +1,102 @@
...
+
+### Redirects
+
+| Redirects   | To    |
+|-------------|----------|
+| `/Customer/Details/{id}`  | `/Customer/Edit/{id}`, `/Case/Details/{id}` |
+
+## Contact Pages
+
+* `/Contact/Details/{id}` (List of Customers handled by a Program Contact)
```
Comment (dougwaldron) [inline docs/Site map.md:48]: Remove.

File: docs/Site map.md [hunk 2]
```diff
@@ -1,22 +1,102 @@
...
+| Redirects   | To   |
+|---------------|----------|
+| `/ActionItem/Details/{id}`  | `/Case/Details/{id}` |
+
+## Maintenance Pages 
+> Maintenance pages available to Site Admin personnel to modifiy Drop Down Menus
+
+### Office Maintenance Pages
+
+* `/Maintenance` (List of Items that can be modified on Site - Action Items and Offices)
```
Comment (dougwaldron) [inline docs/Site map.md:67]: There is also an "Agency" maintenance item. See #62.

File: docs/Site map.md [hunk 3]
```diff
@@ -1,22 +1,102 @@
...
+
+### Office Maintenance Pages
+
+* `/Maintenance/Office` (List of Internal Offices to assign cases to. Can be removed from list here)
+* `/Maintenance/Office/Add` (Add New Office)
+* `/Maintenance/Office/Edit/{id}` (Edit Office properties)
+
+### Action Item Maintenance Pages
+
+* `/Maintenance/ActionItem/Details/{id}` (Details for Action Item. Links to Edit)
```
Comment (dougwaldron) [inline docs/Site map.md:77]: Remove the Details page.

File: docs/Site map.md [hunk 4]
```diff
@@ -1,22 +1,102 @@
...
 * `/Admin/Users/Details/{id}`
 * `/Admin/Users/Edit/{id}` (edit contact info)
 * `/Admin/Users/EditRoles/{id}` (edit roles)
+
+# Menu Bar
+
+* Home (Redirect to Search Page: `/`)
+* Account (Redirect to Account Edit Page: `/Account/Edit/{id}`)
+* Add Customer (Redirect to Add Customer Page: `/Customer/Add`)
+* Add Case (Redirect to Add Case Page: `/Case/Add`)
```
Comment (dougwaldron) [inline docs/Site map.md:101]: Add Case won't be linked from the menu bar. It's only linked from the Customer page.

**Review by dougwaldron — CHANGES_REQUESTED** [review]
Please see the attached comments. Thanks.

**Comment by tom-karasch** [comment]
I think I made all the correct changes. It is a little different page arrangement than I thought.

**Review by tom-karasch — COMMENTED** [review]
Sorry, Doug if it was a bit sloppy, or off a bit from your plan of ASP pages. Thanks for sprucing it up!

**Pull request merged**